.github/workflows/ci-test-xtgeo.yml

```diff
@@ -69,37 +23,28 @@ jobs:
             python-version: 3.6
           - os: windows-latest
             python-version: 3.8
-
     steps:
       - uses: actions/checkout@v1
-
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-
       - name: Install swig (osx)
         if: matrix.os == 'macos-latest'
         run: brew install swig
-
       - name: Install swig (win)
         if: matrix.os == 'windows-latest'
-        run: |
-          (New-Object System.Net.WebClient).DownloadFile("http://prdownloads.sourceforge.net/swig/swigwin-4.0.2.zip","swigwin-4.0.2.zip");
-          Expand-Archive .\swigwin-4.0.2.zip .;
-          echo "$((Get-Item .).FullName)/swigwin-4.0.2" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
-
+        run: "choco install -y --no-progress --allow-empty-checksums -r swig --version=4.0.1"
       - name: Install xtgeo
         run: |
           pip install pip -U
           pip install .
-
       - name: Test with pytest
         run: |
           git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
+          pip install -r requirements/requirements.txt
```

Comment:
Should this not be covered by `pip install .`?

Reply:
Correct (but does not do any harm except some seconds)

Reply:
Well... If we update our `requirements` file, and for some reason hardcode the packages in `setup.py` they can come out of sync, and we don't catch that because we install both places.. It's always possible to come up with some scenario which could throw us off..

Besides, I think we should trust our setup enough to not have redundant commands included.

Reply:
:+1: